apps/pie-microsite/src/_11ty/shortcodes/contentPageImage.js

```diff
@@ -7,7 +7,7 @@ const markdownFilter = require('../filters/markdown');
  * @returns {String} a <figcaption> element containing the caption text, or an empty string if no caption is provided.
  */
 const createCaption = config => (config.caption
-    ? `<figcaption class="c-contentImage-caption">${markdownFilter(config.caption)}</figcaption>`
+    ? `<figcaption class="c-contentImage-caption"><span>${markdownFilter(config.caption, true)}</span></figcaption>`
```

Comment:
More out of interest than anything – what's the span for here? (just as there's no styling in this PR to go along with adding it)

Reply:
The markdown filter returns it without and enclosing tag which meant that the styling would break if it had another html tag rendered as part of the html. I've removed it now as I've round that I can get the markdown filter to return the span so we don't need to apply it to every shortcode.